docs/source/metadata.rst

```diff
@@ -134,14 +134,15 @@ New scopes are created for classes, functions, and comprehensions. Other block
 constructs like conditional statements, loops, and try…except don't create their
 own scope.
 
-There are four different type of scope in Python:
+There are five different type of scope in Python:
+:class:`~libcst.metadata.BuiltinScope`,
```

Comment:
The BuiltinScope is not added successfully in the doc.
https://9611-200896124-gh.circle-artifacts.com/0/doc/metadata.html
You'll need to add it to __init__.py to make it work.
https://github.com/Instagram/LibCST/blob/master/libcst/metadata/__init__.py